Portfolio/src/assets/Variables.scss

```diff
@@ -1,11 +1,23 @@
-// $color-pink-primary: #D4ABB2;
-// $color-black-heading: #1e1e1e;
-// $color-white: #ffff;
-
-// $font-family-primary: "Raleway";
-// $font-size-small: 1.5rem;
-
-// $margin-top-small: 5rem;
-// $margin-top-large: 10rem;
+$margin-top-xsmall: 1rem;
+$margin-top-medium: 5rem;
+$margin-top-large: 10rem;
+$margin-zero: 0;
```

Comment:
Similar to the comment below about text-transform, I dont think margin-zero: 0 adds any value

Reply:
Done